### PR TITLE
Add new pricing route fallback

### PR DESCRIPTION
**Pull request by dospore**
Add fallback if new pricing is stale for whatever reason

### CODE REVIEW

File: src/hooks/useInfoTokens.ts
Comment (chrdevmar) [inline src/hooks/useInfoTokens.ts:60]: this condition will cause no prices to be used if one endpoint fails. we should make it so that if only one succeeds it uses that, and only perform stale checks/overwrite if both are available

File: src/hooks/useInfoTokens.ts
Comment (chrdevmar) [inline src/hooks/useInfoTokens.ts:65]: waiting on confirmation of to approve this line

https://github.com/mycelium-ethereum/swaps-prices/pull/27/files/88dd591144a25f246f73dee5900b3beff2821f52..ec19bad9971e806355b8b06e17085b06fa9114f0#r1019836962

**Review by chrdevmar — CHANGES_REQUESTED** [review]
happy to approve when https://github.com/mycelium-ethereum/swaps-prices/pull/27/files/88dd591144a25f246f73dee5900b3beff2821f52..ec19bad9971e806355b8b06e17085b06fa9114f0#r1019836962 is confirmed

File: src/hooks/useInfoTokens.ts
Comment (dospore) [reply src/hooks/useInfoTokens.ts:65]: confirmed

**Review by chrdevmar — APPROVED** [review]

**Pull request merged**
